examples/k8s_cluster/main.tf

```
@@ -10,9 +10,9 @@ provider "illumio-cloudsecure" {
 }
 
 module "k8s_cluster_dev" {
-  source         = "illumio/cloudsecure/illumio//modules/k8s_cluster"
-  version        = "1.4.3"
-  illumio_region = "aws-us-west-2"
-  name           = "example-release"
-  description    = "Dev cluster in aws-us-west-2"
-}
\ No newline at end of file
+  source           = "illumio/cloudsecure/illumio//modules/k8s_cluster"
+  operator_version = "1.4.3"
```

Ah, we need 2 variables. `version` is the module version.
```suggestion
    version          = "1.4.3"
    operator_version = "v1.0.5"
```